myworld1/web_scrapper.py

```diff
@@ -0,0 +1,95 @@
+import psycopg2
+import requests
+import re
+from bs4 import BeautifulSoup, element
+import os
+        
+# For the credentials mentioned below, you may refer the docker-compose.yml present in myworld .
+db_name = 'member_db'
+db_user = 'postgres'
+db_pass = '123456'
+db_host = 'psql-db'
+db_port = '5432'
+        
+# This will create the connection the to postgres database.
+conn = psycopg2.connect(dbname=db_name, user=db_user, password=db_pass, host=db_host, port=db_port)
+                   
+                    
+def add_row_to_blog(title, author, date, time,content,recommended,path):
+    # This function will add the entry to database
+    sql = """INSERT INTO members_blog (title, release_date, blog_time, created_date,content, author,recommended,path) VALUES (%s, %s::DATE, %s::TIME,  NOW(),%s,%s,%s,%s)"""
+            
+    with conn:
+        with conn.cursor() as curs:
+            time=time.replace('\u202f',"")
```

Comment:
@AnkitaHegde Why are you replacing it?

Reply:
Over there, as we scrape the time we were getting white-spaces which was causing InvalidSyntax error so just replaced that unicode with empty string.